modules/javafx.graphics/src/main/native-glass/mac/GlassMenu.m

```diff
@@ -262,6 +262,22 @@ - (void)_setPixels:(jobject)pixels
         (*env)->CallVoidMethod(env, pixels, jPixelsAttachData, ptr_to_jlong(&image));
         if (image != NULL)
         {
+            jclass pixelsClass = (*env)->FindClass(env, "com/sun/glass/ui/mac/MacPixels");
```

Comment:
Use `[GlassHelper ClassForName:"com.sun.glass.ui.mac.MacPixels" withEnv:env];` instead, 
and then check `pixelsClass` is not null.